### PR TITLE
Improvement/cross device option logic

### PR DESCRIPTION
**Pull request by ethan605**
# Problem

The current implementation of CrossDeviceLink needs improvements for better maintainability.

# Solution

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [n/a] Has the README been updated?
- [n/a] Has the CONTRIBUTING doc been updated?
- [n/a] Has the RELEASE_GUIDELINES been updated?
- [n/a] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [n/a] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [n/a] Have any new automated tests been implemented or the existing ones changed?
- [n/a] Have any new manual tests been written down or the existing ones changed?
- [n/a] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Review by stefaniacardenas — APPROVED** [review]
Would be good to still have a small description for this PR

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1228-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1228-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1228-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1228-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/crossDevice/CrossDeviceLink/index.js [hunk 1]
```diff
@@ -102,8 +133,12 @@ class CrossDeviceLink extends Component {
 class CrossDeviceLinkUI extends Component {
   constructor(props) {
     super(props)
+
     this.state = {
-      currentViewId: 'qr_code',
+      currentViewId: validatesViewIdWithFallback(
+        props.steps.find(({ type }) => type === 'document')?.options
+          ?._initialCrossDeviceLinkView
```
Comment (kopijunkie) [inline src/components/crossDevice/CrossDeviceLink/index.js:140]: Why does this step option property have an underscore before the name?

**Review by kopijunkie — APPROVED** [review]
LGTM. Just have a question about the property name.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1228-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1228-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1228-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**